recipes/pidly/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "pIDLy" %}
+{% set version = "0.2.7" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "6c9e77d710c09de6856431869b0cca2bc8ac2de961af82ac8665b38fc535567e" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+  noarch: python
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy
```

Comment:
You can drop `numpy`, `pexpect` and `scipy` as `build` requirements. Only modules that are imported by the `setup.py` (either directly or as a side-effect) or are listed in `setup_requires` in the `setup.py` are needed for the `build` when using `--single-version-externally-managed`